dev/devicelab/lib/framework/browser.dart

```diff
@@ -576,3 +558,72 @@ int _readInt(Map<String, dynamic> json, String key) {
 
   return jsonValue.toInt();
 }
+
+/// Used by [Chrome.launch] to detect a glibc bug and retry launching the
+/// browser.
+///
+/// Once every few thousands of launches we hit this glibc bug:
+///
+/// https://sourceware.org/bugzilla/show_bug.cgi?id=19329.
+///
+/// When this happens Chrome spits out something like the following then exits with code 127:
+///
+///     Inconsistency detected by ld.so: ../elf/dl-tls.c: 493: _dl_allocate_tls_init: Assertion `listp->slotinfo[cnt].gen <= GL(dl_tls_generation)' failed!
```

Comment:
very nit: is this spaces intentional?

Reply:
Yes, the 4-space indent is treated by dartdoc as a code snippet.

Reply:
thanks! I haven't used this before.